src/components/molecules/footers/FooterLinks.tsx

```diff
@@ -0,0 +1,30 @@
+import { Link } from 'react-router-dom'
+
+type FooterLinksProps = {
+  title: string
+  items: { label: string; link: string }[]
+}
+
+export function FooterLinks({ title, items }: FooterLinksProps) {
+  return (
+    <div
+      style={{
+        display: 'flex',
+        flexDirection: 'column',
+        color: 'white',
+        marginBottom: '10px',
+      }}
+    >
+      <p style={{ margin: '5px 0', fontWeight: 'bold' }}>{title}</p>
```

Comment:
:wrench: Instead of a `<p>` tag, I'd use a <Typography> tag and we can give it the variant we want.